JobRunner/Callback.py

```diff
@@ -27,7 +27,11 @@ def load_prov(self, job_params_file):
             raise ValueError("params in Provenance file isn't a list")
         return job_params
 
-    def run(self):
+    def start(self):
```

Comment:
Can we make this a more descriptive name ? It is not clear what is being started. 

I have forgotten what much of the code does, and from looking at these files it is not obvious what even is a "CallBack" vs a "CallBackServer"

Reply:
self.cbs is no longer used, is that right? if so, it can be removed.

Reply:
Fixed both issues